src/LibraryTest/Library/WebServerTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿namespace Microsoft.IstioMixerPlugin.LibraryTest.Library
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Text.RegularExpressions;
+    using System.Threading.Tasks;
+    using ApplicationInsights;
+    using ApplicationInsights.Channel;
+    using ApplicationInsights.DataContracts;
+    using ApplicationInsights.Extensibility.Implementation;
+    using Google.Protobuf;
+    using Google.Protobuf.WellKnownTypes;
+    using Istio.Policy.V1Beta1;
+    using IstioMixerPlugin.Library;
+    using Tracespan;
+    using VisualStudio.TestTools.UnitTesting;
+    using Uri = System.Uri;
+
+    [TestClass]
+    public class WebServerTests
+    {
+        private string config;
+        private static readonly TimeSpan DefaultTimeout = TimeSpan.FromSeconds(1);
+        private WebServer webServer;
+        [TestInitialize]
+        public void init()
```

Comment:
[minor] Init, in C# we always Pascal case method names. Same below for "cleanup"

Reply:
done